src/dotnet/UI.Blazor/Components/Account/AccountBadgeBase.cs

```diff
@@ -30,17 +28,10 @@ protected override ComputedState<Model>.Options GetStateOptions()
         if (account == null)
             return Model.None;
 
-        var presence = Presence.Unknown;
-        if (ShowPresence)
-            presence = await UserPresences.Get(UserId, cancellationToken);
-
-        return new(account, presence);
+        return new(account);
     }
 
-    public sealed record Model(
-        Account Account,
-        Presence Presence = Presence.Unknown
-    ) {
+    public sealed record Model(Account Account) {
```

Comment:
Record struct?

Reply:
It's less important here though - the other models I asked the same question about are used frequently.